src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {

```

Comment:
There is no need to use the 'Random' instance as a class member unless it is used in multiple methods or its state is important across multiple calls to 'getRandomColor'. If it is only used in this method, you could make it a local variable within the method.